configs/components/curl.rb

```diff
@@ -6,30 +6,21 @@
   pkg.build_requires "openssl"
   pkg.build_requires "puppet-ca-bundle"
 
-  prefix = settings[:prefix]
-  pem_file = "#{settings[:prefix]}/ssl/cert.pem"
-
   if platform.is_windows?
-    pkg.build_requires "pl-zlib-#{platform.architecture}"
     pkg.build_requires "runtime"
 
-    pkg.environment "PATH" => "#{settings[:gcc_bindir]}:$$PATH"
+    pkg.environment "PATH" => "$$(cygpath -u #{settings[:gcc_bindir]}):$$PATH"
     pkg.environment "CYGWIN" => settings[:cygwin]
-    pkg.environment "CC" => settings[:cc]
```

Comment:
There is no longer a need to set the compiler environment? (i'm mostly just curious)